package.json

```diff
@@ -111,6 +111,7 @@
     "verbose": true,
     "automock": false,
     "setupFiles": [
+      "./src/shim.js",
```

Comment:
Can we just merge this into `setupJest.js`?

Reply:
Unfortunately that doesn't get rid of the warning :( 
https://github.com/facebook/jest/issues/4545#issuecomment-332762365